docs/src/parameter_distributions.md

```diff
@@ -21,119 +26,106 @@ dict_1 = Dict("distribution" => distribution_1, "constraint" => constraint_1, "n
 dict_2 = Dict("distribution" => distribution_2, "constraint" => constraint_2, "name" => name_2)
 prior = ParameterDistribution( [dict_1, dict_2] )
 ```
+
 We provide many examples in the package `examples/`, the unit tests found in `test/ParameterDistributions/runtests.jl`, and in the documentation below.
 
-# A simple example:
-Task: We wish to create a prior for a one-dimensional parameter. Our problem dictates that this parameter is bounded between 0 and 1. Prior knowledge dictates it is around 0.7. The parameter is called `point_seven`.
+### Constrained\_gaussian constructor
 
-Solution: We should use a Normal distribution with the predefined "bounded" constraint.
+The constructors described above can be used to specify arbitrary prior distributions and constraints. However, the anticipated use case for EnsembleKalmanProcesses is one where inference is not strongly prior-dominated: i.e., the user only needs (or is only able to) specify the physical and likely ranges of prior parameter values at a rough, qualitative level.
```

Comment:
`EnsembleKalmanProcesses`? in quotes?

Reply:
Remove the phrase "strongly prior dominated", perhaps instead just say "where prior information is qualitative, and exact distributions of the priors are unknown."

Reply:
Done in [9b09cea](https://github.com/CliMA/EnsembleKalmanProcesses.jl/pull/161/commits/9b09cea7032cefb92cd0332b9656b924152356c1)